lib/bulk_data_test_kit/v2.0.0/client/bulk_data_client_test_suite.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative '../../version'
+
+require_relative './endpoints/delete'
+require_relative './endpoints/kick_off'
+require_relative './endpoints/output'
+require_relative './endpoints/status'
+
+require_relative './groups/export'
+require_relative './groups/delete'
+
+module BulkDataTestKit
+  module BulkDataV200
+    module Client
```

Comment:
Not sure if this is something that should be changed, but I am thinking to be consistent with the rest of the bulk data test suites, there is no need to add the Client module here

Reply:
I think we want to separate out the client tests in some way, right? Do you mean move `Client` up one level e.g. have a `module BulkDataV200Client`?

Reply:
Yeah that makes sense, I've made this change.